src/main/java/com/solutions/shop/controller/CustomerController.java

```diff
@@ -0,0 +1,25 @@
+package com.solutions.shop.controller;
+
+import com.solutions.shop.dto.CustomerDto;
+import com.solutions.shop.service.CustomerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class CustomerController {
+    @Autowired
+    private CustomerService customerService;
+
+    @PostMapping("/customer")
+    public ResponseEntity<?> createCustomerInfo(@RequestBody CustomerDto customer) {
+        customerService.addInfoForUser(customer.getUserId(),
+                customer.getFirstName(),
+                customer.getLastName(),
+                customer.getAddress());
+        return new ResponseEntity<>(HttpStatus.CREATED);
```

Comment:
Incorrect result handling

Reply:
return new ResultDTO("Information saved successfully!");
        
        public ResponseEntity<?> -> public ResponseEntity<String>
        
        return new ResponseEntity<>(HttpStatus.CREATED); -> return new ResponseEntity<String>(HttpStatus.CREATED, <text from service>);

Change to success text response